src/ape/api/address.py

```diff
@@ -79,6 +81,30 @@ def __str__(self) -> str:
         """
         return self.address
 
+    def __call__(self, **kwargs) -> "ReceiptAPI":
+        """
+        Call this address directly.
+
+        Args:
+            **kwargs: Transaction arguments, such as ``sender`` or ``data``.
+
+        Returns:
+            :class:`~ape.api.transactions.ReceiptAPI`
+        """
+
+        converted_kwargs = _convert_kwargs(kwargs, self.conversion_manager.convert)
```

Comment:
@fubuloubu I feel like I am missing something. This seems to work, but it doesn't seem like there is any real use of the FallbackABI or the ReceiveABI when it comes to this, like that all happens on the EVM side.

Is there anything missing in this PR around the 5 different scenarios of FallbackABI / ReceiveABI combinations or any missing test cases I should add?

I think this PR allows the fallback and receive methods to at least be testable now, which is all I needed here for moving forward with Coverage.

Reply:
After the transaction is created, if the object is a contract instance and has a fallback or receive function we can do some additional validation on the scenarios that are supported.

That might be better off in the ethereum plugin to be honest 🤔

Reply:
I think we could put it in a validator from one of the transaction classes defined in `ape_ethereum.transactions`.

Reply:
Hmm, maybe could subclass `ContractInstance.__call__` to do this check before using this call instead

Reply:
ya i was thinking that too, but that wouldn't solve the evm-ness entirely, but maybe for now that is all we do and if some zk thing comes up we incorporate an `ape.api`